app/assets/javascripts/social.js

```diff
@@ -0,0 +1,19 @@
+$(window).load(function() {
+
+  ///// Social media buttons
+  if ($('.shareSave').length) {
+    (function(d, s, id) {
```

Comment:
I'm not sure that the closures here and on line 13 are necessary for privacy / variable scope reasons, considering that this is within an anonymous function given to `load()` above.  Do you have a position in favor of their existing here?